client/components/Stage.jsx

```diff
@@ -0,0 +1,45 @@
+import {
+  Button,
+  Card,
+  CardGroup,
+  CardImg,
+  Col,
+  Container,
+  Input,
+  Row,
+  Dropdown,
+  DropdownItem,
+  DropdownToggle,
+  DropdownMenu,
+  Jumbotron,
+  Alert,
+  Carousel,
+  CarouselIndicators,
+  CarouselControl,
+  CarouselCaption,
+  CarouselItem,
+  FormGroup,
+  Label
+} from "reactstrap";
+import "../public/styles/stage.scss";
+import "../public/styles/style.scss";
+import { Head } from ".";
+export default function(props) {
+  console.log("props.value");
+  console.log(props);
+  return (
+    <>
+      <Head />
```

Comment:
components in the `components` folder don't need a `Head`, only those in pages do